x/crosschain/keeper/msg_tss_voter.go

```diff
@@ -129,6 +129,32 @@ func (k msgServer) UpdateTssAddress(goCtx context.Context, msg *types.MsgUpdateT
 	return &types.MsgUpdateTssAddressResponse{}, nil
 }
 
+func (k msgServer) MigrateTssFunds(goCtx context.Context, msg *types.MsgMigrateTssFunds) (*types.MsgMigrateTssFundsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	if msg.Creator != k.zetaObserverKeeper.GetParams(ctx).GetAdminPolicyAccount(observerTypes.Policy_Type_group1) {
```

Comment:
Can we add it in the Notion page for group policies?
I also think this should be group 2 if it implies movement of funds

Reply:
@kingpinXD do you have any thought about using group 2?

Reply:
Yeah. Policy group2 would be safer